File: leet_2032_2_outta_3.cc
class Solution {
public:
    vector<int> twoOutOfThree(vector<int>& nums1, vector<int>& nums2, vector<int>& nums3)
    {
        set<int>    s;
        for (int n: nums1)
        {
            // if (find(nums2.begin(), nums2.end(), n) != nums2.end() || find(nums3.begin(), nums3.end(), n) != nums3.end())
            if (count(nums2.begin(), nums2.end(), n) || count(nums3.begin(), nums3.end(), n))
            s.insert(n);
        }
        for (int n: nums2)
        {
            // if (find(nums3.begin(), nums3.end(), n) != nums3.end())
            if (count(nums3.begin(), nums3.end(), n))
                s.insert(n);
        }
        return vector<int>(s.begin(), s.end());
    }
};

File: leet_2032_2_outta_3.py
class Solution:
    def twoOutOfThree(self, nums1: List[int], nums2: List[int], nums3: List[int]) -> List[int]:
        s = set()
        for n in nums1:
            if n in nums2 or n in nums3:
                s.add(n)
        for n in nums2:
            if n in nums3:
                s.add(n)
        return list(s)
